bin/GameIndex.dbf

```diff
@@ -2023,20 +2012,16 @@ Region = PAL-Unk
 Serial = SCES-52677
 Name   = Network Access Disc & Hardware - Online Arena
 Region = PAL-M7
-Compat = 5
+Compat = 5	// With GSdx, SW-mode only for HOA.
```

Comment:
Can you remove the comment here ? I don't like it being on the compatibility line.